include/OpenSpaceToolkit/Astrodynamics/Trajectory/State.hpp

```diff
@@ -33,6 +33,8 @@ using ostk::physics::time::Instant;
 class State
 {
    public:
+    State(const Instant& anInstant, const Shared<const Frame>& aFrameSPtr, const VectorXd& aCoordinates);
```

Comment:
Nit:
```suggestion
    State(const Instant& anInstant, const VectorXd& aCoordinates, const Shared<const Frame>& aFrameSPtr);
```
Re order to maintain the concept of `time, coordinates, ...`